gnovm/pkg/gnolang/type_check.go

```diff
@@ -279,6 +279,20 @@ func checkValDefineMismatch(n Node) {
 	panic(fmt.Sprintf("assignment mismatch: %d variable(s) but %d value(s)", numNames, numValues))
 }
 
+func assertValidAssignRhs(store Store, last BlockNode, exps Exprs) {
```

Comment:
can you also put this(along with some other func like `checkValDefineMismatch`... into // misc?

Reply:
right moved here: https://github.com/gnolang/gno/pull/2896/commits/7d935f616323f9707d899c449ed4661546aeb392